crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -262,6 +262,7 @@ def status
                 "pending" => 0,
                 "unready" => 0,
                 "building" => 0,
+                "crowbar-upgrade" => 0,
```

Comment:
Then you have to fix invalid javascript like `val.status.crowbar-upgrade` as this results in `val.status.crowbar - upgrade` (upgrade is an undefined variable)